packages/manager/apps/container/src/container/nav-reshuffle/sidebar/SidebarLink.tsx

```diff
@@ -45,7 +46,15 @@ const SidebarLink: React.FC<ComponentProps<SidebarLinkProps>> = ({
       onClick={handleOnClick}
       id={id}
     >
-      <span> {t(isShortText ? node.shortTranslation : node.translation)}</span>
+      {isShortText ?
+        <ShortSidebarLinkTooltip content={t(node.translation)}>
+          {t(node.shortTranslation)}
+        </ShortSidebarLinkTooltip>
+        :
+        <span>
+          {t(node.translation)}
+        </span>
+      }
```

Comment:
as you've created a component named ShortSidebarLinkTooltip, it means it's only for the sidebar (not a generic one). 
In that case, why not creating an entire component to deal with ?

```
{ isShortText ? <SidebarTooltipLink tooltip={content} text={text} /> : <span>
          {t(node.translation)}
        </span>
}
```

Reply:
I hadn't thought of that, I've just created it. Thank you for your feedback.